contracts/Tokens/VTokens/VToken.sol

```diff
@@ -1371,6 +1365,82 @@ contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
         return (uint(Error.NO_ERROR), actualRepayAmount);
     }
 
+    /**
+     * @dev Allows a borrower to swap his debt between stable and variable mode, or vice versa
+     * @param rateMode The rate mode that the user wants to swap to
+     **/
+    function swapBorrowRateMode(uint256 rateMode) external {
+        /* Fail if swapBorrowRateMode not allowed */
+        comptroller.swapBorrowRateModeAllowed(address(this));
+
+        accrueInterest();
+
+        /* Verify market's block number equals current block number */
+        if (accrualBlockNumber != getBlockNumber()) {
```

Comment:
We can remove this statement

Reply:
Fix in [Commit](https://github.com/VenusProtocol/venus-protocol/pull/218/commits/47722840b0a50747600bc29da312e66bf352e3a2)